<launch>
    <arg name="camera_frame"      default="external_camera_link"/>
    <arg name="marker_frame_ext"      default="marker26"/>   
    <arg name="marker_frame_eye"      default="marker26b"/>   
    <arg name="robot_base_frame"  default="world"/>

    <!-- File will be loaded if calibrate_from_file is true-->
    <arg name="load_calib_file"    default="$(find boris_tools)/calib_data/external_eye_calibration.npy"/> 
    <arg name="calibrate_from_file"    default="true"/>
    
    <arg name="save_calib_file"    default="external_eye_calibration.npy"/>
    

    <node pkg="boris_tools" type="external_eye_calibrator.py" name="boris_external_eye_calibrator" output="screen">
        <param name="calibrate_from_file" value="$(arg calibrate_from_file)"/>
        <param name="load_calib_file" value="$(arg load_calib_file)"/>
        <param name="save_calib_file" value="$(arg save_calib_file)"/>
        <param name="camera_frame"      value="external_camera_link"/>
        <param name="marker_frame_ext"      value="$(arg marker_frame_ext)"/>
        <param name="marker_frame_eye"      value="$(arg marker_frame_eye)"/>
        <param name="robot_base_frame"  value="$(arg robot_base_frame)"/>
    </node>

</launch>

